apps/docs/pages/guides/functions/troubleshooting.mdx

```diff
@@ -34,6 +34,31 @@ There are two debugging tools available: Invocations and Logs. Invocations shows
 - Search the [Edge Runtime](https://github.com/supabase/edge-runtime) and [CLI](https://github.com/supabase/cli) repos for the error message, to see if it has been reported before.
 - If the output from the commands above does not help you to resolve the issue, please open a support ticket via the Supabase Dashboard (by clicking the "Help" button at the top right) and include all output and details about your commands.
 
+# Advanced Techniques
+
+## Finding Bundle Size
+
+To find the bundle size of a function, run the following command locally:
+
+```bash
+deno info /path/to/function/index.ts
+```
+
+Look for the "size" field in the output which represents the bundle size of the function. If the function is part of a larger application, consider examining the bundle size of the specific function independently.
+
+## Analyze Dependencies
+
+Review the dependencies listed in the output of the `deno info` command. Pay attention to any significantly large dependencies, as they can contribute to increased bundle size and potential boot time issues.
```

Comment:
lets change "Make sure to also check the import maps" -> "When running `deno info` make sure to provide the correct path of the import map if you use one.